src/Tests/WorkspaceTests.cs

```diff
@@ -80,6 +80,26 @@ public void ProjectReferencesWorkspace()
             Assert.IsTrue(operations.Where(o => o.FullName == "Tests.ProjectReferences.ProjectA.RotateAndMeasure").Any());
         }
 
+        [TestMethod]
+        public void ProjectReferencesWorkspaceNoAutoLoad()
+        {
+            // Loading this workspace should fail because the .csproj does not specify <IQSharpLoadAutomatically>,
+            // which prevents the code that depends on Workspace.ProjectReferences.ProjectB from compiling correctly.
+            var ws = Startup.Create<Workspace>("Workspace.ProjectReferences.ProjectA");
+            ws.Reload();
+            Assert.IsTrue(ws.HasErrors);
+
+            // Loading this workspace should succeed, and its Q# operations should be available, but the .csproj
+            // reference should not be loaded because the .csproj does not specify <IQSharpLoadAutomatically>.
+            ws = Startup.Create<Workspace>("Workspace.ProjectReferences.ProjectB");
```

Comment:
It would be nice to add a test for a project that explicitly disable iQ# loading (i.e. has `IQSharpLoadAutomatically` set to `false`)